### PR TITLE
LVM-431 / LVM-432: Update tutor form fields and views.

### PR DESCRIPTION
**Pull request by nabeel-**
* Cleans up the tutor form view
* Adds sections and scroll spy to form (using [bootstrap's scrollspy](http://getbootstrap.com/javascript/#scrollspy))
* Cleans up and restyles tutor show view

TODO:
* before merging, fix tags link by rebasing after #88 is merged.

https://cs5500-jira.ccs.neu.edu/browse/LVM-431
https://cs5500-jira.ccs.neu.edu/browse/LVM-432

screen shot 2017-03-23 at 2 26 08 am

### CODE REVIEW

**Comment by zack** [comment]
Would be nice if we could get 'edit' and 'delete' on the same line for comments. possibly even just with icons (pencil and X). Also, tags link currently goes to comments.

**Comment by zack** [comment]
v nice though

File: app/assets/stylesheets/application.scss [hunk 1]
```diff
@@ -127,12 +131,34 @@
...
+  color: #6d6d6d;
+}
+
+.nav-pills > li.active > a,
+.nav-pills > li.active > a:hover,
+.nav-pills > li.active > a:focus {
+  border-left: 2px solid #8882d8;
+  color: #8882d8;
+  background-color: white;
+}
```
Comment (zack) [inline app/assets/stylesheets/application.scss]:
```
nav-pills > li.active {
  a {
    color: #6d6d6d;
  }
  a, a:hover, a:focus {
    border-left: 2px solid #8882d8;
    color: #8882d8;
    background-color: white;
  }
}
```

**Comment by zack** [comment]
Curious why you're using overwhelmingly phone-style bootstrap classes (`col-xs`)

**Comment by nabeel-** [comment]
@zy I'll update with the comments you mentioned.

I used `xs` classes in some places where I didn't want the columns to stack no matter how small the screen size got.

**Review by zack — APPROVED** [review]

**Comment by zack** [comment]
Looking forward to roberta's feedback on this view.

File: app/models/tutor.rb [hunk 2]
```diff
@@ -48,7 +47,8 @@ class Tutor < ApplicationRecord
                            allow_blank: true
   validates :other_phone,  format: { with: VALID_PHONE_REGEX },
                            allow_blank: true
-  validates :smartt_id,    format: { with: VALID_SMARTT_REGEX }
+  validates :smartt_id,    format: { with: VALID_SMARTT_REGEX },
+                           allow_blank: true
```
Comment (xingyif) [inline app/models/tutor.rb:51]: wait, we shouldn't allow smartt_id to be blank, right?

Comment (nabeel-) [reply app/models/tutor.rb:51]: `smartt_id` is not known when the tutors or students are entered into the system. It's only given to the coordinators after the first monthly cycle of data entry into smart.

**Pull request merged**